roles/openstack-stack/templates/heat_stack_server.yaml.j2

```diff
@@ -154,6 +159,7 @@ resources:
         host-type: { get_param: type }
         sub-host-type:    { get_param: subtype }
         node_labels: { get_param: node_labels }
+      scheduler_hints: { get_param: scheduler_hints }
```

Comment:
indentation fix?

Reply:
@bogdando thanks for pointing that out! It's actually correct; node_labels is part of the metadata entry (you can see it if you look at the full file).